### DISCUSSION TITLE
Prefixing a class after pseudoclass with '!' is causing compilation error in Sass

### DISCUSSION THREAD

**Discussion by ccrsxx** [tailwindlabs/tailwindcss#9727] [Help] [answered]
Hi.

I want to override a dark theme with a custom components class, so I need to put important '!' after the pseudoclass.

![image](https://user-images.githubusercontent.com/55032197/199774734-66194690-3245-4924-8a0a-eb23d1f1830a.png)

But it causes compilation errors. 

![image](https://user-images.githubusercontent.com/55032197/199776253-44ebc687-3cf6-40e8-a5a8-d686f5843364.png)

I tried using CSS to ensure there wasn't any syntax error, and it worked there. So the problem is in the Sass.

Is there a way to fix this compilation error in Sass without changing the file to CSS?

**Comment by wongjn** [comment, marked as answer]
You will run into some conflicts with Sass from time to time. For your particular you could try interpolating the problem string:
```scss
.accent-tab {
  @apply #{'focus-visible:!ring-main-accent/60'};
}
```

**Reply by ccrsxx** [reply]
Thanks for the answer, It works. I'll consider using CSS again in the future.
